### PR TITLE
add autopilot scenario

### PR DESCRIPTION
**Pull request by wallnerryan**
- Adds new scenario for AutoPilot

cc @harsh-px was able to run through this, feedback is integrated. 

Try out current iteration at https://katacoda.com/wallnerryan

replaces old PR with squashed commits https://github.com/portworx/katacoda-scenarios/pull/54

### CODE REVIEW

File: px-k8s-auto-resize/6.fill-and-resize-volume.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+pgbench -i -s 50 pxdemo
+```{{execute T1}}
+
+* Note that once the test completes, **AutoPilot will make sure the usage remains above 20% for about 30 seconds before triggering the rule.** Click ```exit```{{execute T1}} top exit from the pod shell before proceeding.
+
+### Step: Check to see if the rule was triggered
+
+We can retrieve events by using the `kubectl get events` and filtering for `AutoPilotRule` events that match our use case. Note, that AutoPilot delays the rule being triggered immediately, so make sure to **hang tight and see the rule get triggered if you dont see it right away**.
```
Comment (harsh-px) [inline px-k8s-auto-resize/6.fill-and-resize-volume.md]: can be renamed as 

`that AutoPilot delays the rule from being triggered immediately to ensure that the conditions stablize`

**Comment by wallnerryan** [comment]
thanks @harsh-px just made the text edit. 👍

**Comment by wallnerryan** [comment]
@harsh-px Made a few more edits besides your text edit, mostly improved the flow and makes sure autopilot gets installed properly with retries in the background, i saw on a couple runs it would install if the CRD wasn't ready etc so should be good now.

**Comment by harsh-px** [comment]
Merging this so we can run this from the portworx account.

**Pull request merged**